### PR TITLE
Add manifest.json file generation

### LINKED ISSUE #11: Introduce asset revisioning
http://symfony.com/blog/new-in-symfony-3-3-manifest-based-asset-versioning

### PR DESCRIPTION
**Pull request by Atala**
Seems to work ok

manifest.json will be used only in production : 
 - `NODE_ENV` should be set to `production`
 - `app/config/config_prod.yml` is used 

Fix #11

### CODE REVIEW

**Comment by alexsegura** [comment]
Looks good 👍

Composer still fails on Travis, I'm checking.

**Comment by alexsegura** [comment]
I have fixed the Composer problem, but now there's a segmentation fault :trollface: 

Can't understand why on Travis build succeeded however.

File: js/api/tracking/index.js [hunk 1]
```diff
@@ -18,13 +18,11 @@ var envMap = {
 
 var ConfigLoader = require('../ConfigLoader');
 
-var env = 'production' || 'development';
```
Comment (Atala) [inline js/api/tracking/index.js]: :( - thanks for the fix, shouldn't have commit this

File: webpack.config.js [hunk 2]
```diff
@@ -4,27 +4,28 @@ var ExtractTextPlugin = require("extract-text-webpack-plugin");
...
+    'css/tracking': './assets/css/tracking.scss'
+    'js/bootstrap': 'bootstrap',
+    'js/cart': './js/app/cart/index.jsx',
+    'js/homepage': './js/app/homepage/index.js',
+    'js/order-payment': './js/app/order/payment.js',
+    'js/order-tracking': [ 'whatwg-fetch', './js/app/order/tracking.jsx' ],
+    'js/profile-deliveries': './js/app/profile/deliveries.js',
+    'js/restaurant-form': './js/app/restaurant/form.jsx',
+    'js/delivery-form': './js/app/delivery/form.jsx',
+    'js/tracking': './js/app/tracking/index.jsx',
```
Comment (Atala) [inline webpack.config.js]: Looks nicer indeed, I wasn't sure if it was ok to have a "path" in the entry key.

**Comment by Atala** [comment]
I tried to [remove xdebug](https://docs.travis-ci.com/user/languages/php/#Disabling-preinstalled-PHP-extensions) but it didn't work. Both build failed on same command.

**Comment by Atala** [comment]
The issue was with doctrine cache - I think we can merge now - what do you think @alexsegura ?

File: webpack.config.js (hunk 2 above)
Comment (alexsegura) [inline webpack.config.js]: Not sure either 😐

**Comment by alexsegura** [comment]
I rebased a bit, but now the build never ends 🤔

**Comment by alexsegura** [comment]
WTF? Now it works, when I remove the `-f progress` option from Behat. 

Let's keep this for now, maybe it's a Travis bug??

**Comment by alexsegura** [comment]
Merged. Thanks a lot, deployments will be easier now.

**Pull request merged**